packages/klevu-ui/src/components/klevu-infinite-scroll/klevu-infinite-scroll.tsx

```diff
@@ -0,0 +1,93 @@
+import { Component, Element, Event, EventEmitter, Host, Prop, Watch, h } from "@stencil/core"
+
+/**
+ * Component that triggers event when intercepted on scroll of page.
+ *
+ * @prop infiniteScrollPauseThreshold - The number of pages after which triggers infiniteScrollingPaused event.
+ *                                      Listen to this event to allow further loading on user input.
+ * @prop enabled - Whether infinite scrolling is enabled
+ * @event loadMore - Event emitted when infinite scroll element is intercepted
+ * @event infiniteLoadingPaused - Event emitted when infinite loading reaches a multiple of infiniteScrollPauseThreshold
+ */
+@Component({
+  tag: "klevu-infinite-scroll",
```

Comment:
Name of this component should be `klevu-util-infinite-scroll`

Reply:
renamed